### ISSUE TITLE
Dismembering both legs seems to have no negative impact.

### ISSUE THREAD

**Issue by Greataims** [tgstation/tgstation#41377]
Client version-Latest Beta
## Reproduction:

If you dismember both of your legs and repair the damage to the chest with a bruise pack, you are able to float, running around at normal speed and walking on glass does not trigger a sound. Any dismemberment works for this.

**Comment by k-k-karma** [comment]
oh my god lmao

**Comment by Greataims** [comment]
> oh my god lmao

Yeah, and I figured out after I posted this that it also silences footsteps, which is genuinely beneficial.

**Comment by ShizCalev** [comment]
extensions of this; #41332 #41156
